kotsadm/web/src/components/LicenseUploadProgress.jsx

```diff
@@ -52,6 +55,11 @@ class LicenseUploadProgress extends React.Component {
       </div>
     );
 
+    const hasError = this.state.installStatus === "upload_error";
+    if (hasError) {
+      this.props.onError(this.state.currentMessage);
```

Comment:
is this function passed to all instances of this component? might be a good idea to check if the function `onError` is defined first.